packages/payment-widget/vite.wc.config.ts

```diff
@@ -0,0 +1,42 @@
+import { defineConfig } from "vite";
+import { svelte } from "@sveltejs/vite-plugin-svelte";
+import { nodePolyfills } from "vite-plugin-node-polyfills";
+
+export default defineConfig({
+  define: {
+    global: "globalThis",
+  },
+  plugins: [
+    nodePolyfills({
+      include: ["buffer", "crypto"],
+      exclude: ["http", "stream", "zlib", "assert"],
+    }),
```

Comment:
We removed polyfills from the invoicing template components. Are you sure they're required here?

Reply:
Yes, I tried to get it to work without them, but it did not work.

I added only minimal polyfills to make it work and it did not effect the bundle size.